.slugignore

```diff
@@ -0,0 +1,5 @@
+Dockerfile
```

Comment:
These files should always be ignored from the final .slug. See https://devcenter.heroku.com/articles/slug-compiler#ignoring-files-with-slugignore